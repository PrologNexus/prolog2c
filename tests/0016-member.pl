nl :- foreign_call(write_char(10)).

display(X) :- foreign_call(basic_write(X)).

main :-
	member(X, [1,2,3]), display(X), nl, fail.
main :-
	member(2, [1,2,3]), display('yes'), nl,
	(member(3, [1,2]); display('no'), nl),
	member(1, [Y|_]), display(Y), nl.

member(X, [X|_]).
member(X, [_|R]) :- member(X, R).
