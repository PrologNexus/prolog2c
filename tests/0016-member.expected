1
2
3
yes
[1|_16]
